### ISSUE TITLE
Cannot find name TransferableCaptureBundle, AnalyticsBundle, FMP4Segment

### ISSUE THREAD

**Issue by Gopal6696** [aws-samples/amazon-ivs-player-web-sample#54]
**Project Details**
Angular CLI: 13.3.3
Node: 14.16.1
Package Manager: npm 6.14.12
OS: win32 x64

**package info**
"amazon-ivs-player": "^1.11.0",
"@types/video.js": "^7.3.42",

**Error on ng run or build**
Cannot find name 'TransferableCaptureBundle'.
Cannot find name 'AnalyticsBundle'.
Cannot find name 'FMP4Segment'.

**location**
node-modules/amazon-ivs-player/dist/index.d.ts

**file name**
index.d.ts
![error 1](https://user-images.githubusercontent.com/81372958/180760260-47f2a3c4-05f6-4f2d-97c5-8a43c9556e9c.jpg)

**Comment by alexmdodge** [comment]
@Gopal6696 Thanks for your report. I'll bring this up with the team and investigate.

**Comment by baodtg** [comment]
I faced the same issue, do you guys know any work around to fix this while waiting for aws team to fix it ?

**Comment by nilsabdi** [comment]
> I faced the same issue, do you guys know any work around to fix this while waiting for aws team to fix it ?

Running into the same error on 1.12.0, reverting to 1.10.0 has solved it for me. Althought, looking forward to being able to upgrade to the latest version once this is fixed.

**Comment by johnBartos** [comment]
Hey, sorry for the bug. This should be fixed in 1.14.0. In the future we'll add code checks to make sure we don't introduce unexpected type changes. As a workaround, you can try copying the types we ship in `node_modules` to your source directory, and then manually deleting `CaptureEventMap`. You can then import the types like a regular ES6 import.

```
src/ivs-types.ts

...

import { PlayerEventType } from './ivs-types'
```

**Comment by johnBartos** [comment]
Update, we actually fixed this in 1.13.0, which was just released. Please try it out!
